Makefile

```diff
@@ -669,6 +673,21 @@ api/%.swagger.json: api/%.proto third_party/ build/toolchain/bin/protoc$(EXE_EXT
 		-I $(REPOSITORY_ROOT) -I $(PROTOC_INCLUDES) \
 		--swagger_out=logtostderr=true,allow_delete_body=true:$(REPOSITORY_ROOT)
 
+api/api.md: third_party/ build/toolchain/bin/protoc-gen-doc$(EXE_EXTENSION)
```

Comment:
lets go over the output tomorrow and see if the the autogen output is good enough - else we could use this tool one time to generate a starting point and then keep it updated manually.

Reply:
The output has a PR in the open-match-docs repo

Reply:
Looks pretty good. Looks like agones uses this too? Lets bake this in to our steps.